src/System.Runtime.Extensions/src/System/IO/BufferedStream.cs

```diff
@@ -1246,6 +1246,30 @@ public override void SetLength(long value)
             _stream.SetLength(value);
         }
 
+        public override void CopyTo(Stream destination, int bufferSize)
```

Comment:
@stephentoub I think I added them in a previous PR a while back, but I'll check to confirm. Admittedly I didn't think about tests until now.